_data/projects/rockpaperscissor.yml

```
@@ -0,0 +1,12 @@
+name: Rock, Paper & Scissor
+desc: A simple fully responsive rock,paper and scissors game developed using HTML,CSS & JAVASCRIPT.
+site: https://rock-paper-scissors-game0302.netlify.app/
+tags:
+  - hacktoberfest
+  - javascript
+  - up-for-grabs
+  - front-end-development
+  - open-source
+hacktoberfest:
```

:art: this needs to be updated to address that CI warning

```suggestion
upforgrabs:
```